content/en/docs/concepts/services-networking/service.md

```diff
@@ -150,90 +189,129 @@ Every node in a Kubernetes cluster runs a `kube-proxy`. `kube-proxy` is
 responsible for implementing a form of virtual IP for `Services` of type other
 than [`ExternalName`](#externalname).
 
-In Kubernetes v1.0, `Services` are a "layer 4" (TCP/UDP over IP) construct, the
-proxy was purely in userspace.  In Kubernetes v1.1, the `Ingress` API was added
-(beta) to represent "layer 7"(HTTP) services, iptables proxy was added too,
-and became the default operating mode since Kubernetes v1.2. In Kubernetes v1.8.0-beta.0,
-ipvs proxy was added.
+### Why not use round-robin DNS?
+
+A question that pops up every now and then is why Kubernetes has uses proxying
```

Comment:
i'd reword `why Kubernetes has uses proxying to direct incoming traffic to backends` to

`why Kubernetes need to proxy the incoming direct traffic to backends`

Reply:
I reworded this.